artview/components/limits.py

```diff
@@ -207,6 +207,6 @@ def limits_dialog(limits, cmap, name):
         limits['xmax'] = float(ent_xmax.text())
         limits['ymin'] = float(ent_ymin.text())
         limits['ymax'] = float(ent_ymax.text())
-        print limits
+        print(limits)
```

Comment:
we still keep adding prints for debug, and those are always python 2, as I still use 2.7 as default. This will keep happening, just accept it, there is no reason to get distressed